debian/clearwater-config-manager.links

```diff
@@ -1,6 +1,5 @@
-/usr/share/clearwater/clearwater-config-manager/scripts/check_config_sync /usr/bin/cw-check_config_sync
-/usr/share/clearwater/clearwater-config-manager/scripts/upload_shared_config /usr/bin/cw-upload_shared_config
```

Comment:
Just double checking - we really don't need `cw-upload_shared_config` anymore, do we?

Reply:
We've already deleted the `upload_shared_config` script, so that link is currently broken. Might as well get rid of it.